libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -109,14 +110,45 @@ uint16_t AP_Arming::get_enabled_checks()
     return checks_to_perform;
 }
 
+bool AP_Arming::check_enabled(const enum AP_Arming::ArmingChecks check) const
+{
+    if (checks_to_perform & ARMING_CHECK_ALL) {
+        return true;
+    }
+    if (checks_to_perform & ARMING_CHECK_NONE) {
+        return false;
+    }
+    return (checks_to_perform & check);
+}
+
+MAV_SEVERITY AP_Arming::check_severity(const enum AP_Arming::ArmingChecks check) const
+{
+    if (check_enabled(check)) {
+        return MAV_SEVERITY_CRITICAL;
+    }
+    return MAV_SEVERITY_NOTICE;
+}
+
+void AP_Arming::check_failed(const enum AP_Arming::ArmingChecks check, bool report, const char *fmt, ...)
+{
+    if (!report) {
+        return;
+    }
+    char taggedfmt[MAVLINK_MSG_STATUSTEXT_FIELD_TEXT_LEN];
+    snprintf(taggedfmt, sizeof(taggedfmt), "Arming: %s", fmt);
```

Comment:
I'm not sure everyone will like the Arming text being fixed, instead of the PreArm vs Arm. I don't have issues with it, but devs should be made aware of this change.